chainerx_cc/chainerx/python/error.cc

```diff
@@ -20,6 +20,16 @@ void InitChainerxError(pybind11::module& m) {
     py::register_exception<NotImplementedError>(m, "NotImplementedError");
     py::register_exception<GradientError>(m, "GradientError");
     py::register_exception<GradientCheckError>(m, "GradientCheckError");
+
+    py::register_exception_translator([](std::exception_ptr p) {
```

Comment:
Why not `py::register_exception`?

Reply:
I did not use `py::register_exception` because I don't want to create a new Python exception class.